### PR TITLE
Initial Instructions After Tutorial

### LINKED ISSUE #700: After skipping tutorial interface, display "Let's get started"
The "Let's get started" pop-up does not display after choosing to skip the tutorial on tutorial interface. 
image

### LINKED ISSUE #730: First mission messages should show up after onboarding for a new user
As a new user, right after onboarding, the user should be shown the first mission pop-ups - if they are in their first mission. Currently, they don't appear.

### PR DESCRIPTION
**Pull request by sagec108**
Resolves #730 by displaying mission instructions after user completes tutorial. The pop-ups only show at the start of the first mission.

This PR also partially resolves #700 by displaying the mission instructions after the user clicks "skip the tutorial" without previously completing a mission.

### CODE REVIEW

File: public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js [hunk 1]
```diff
@@ -475,7 +475,15 @@ function Onboarding(svl, actionStack, audioEffect, compass, form, handAnimation,
...
-        modalMission.setMissionMessage(mission, neighborhood);
+        if (missionContainer.isTheFirstMission()) {
+            svl.initialMissionInstruction = new InitialMissionInstruction(svl.compass, svl.map,
+                svl.neighborhoodContainer, svl.popUpMessage, svl.taskContainer, svl.labelContainer);
+            modalMission.setMissionMessage(mission, neighborhood, null, function () {
+                svl.initialMissionInstruction.start(neighborhood);
+            });
+        }else{
+            modalMission.setMissionMessage(mission, neighborhood);
+        }
```
Comment (manaswis) [inline public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js]: Try this code with registered users, it shows the mission box twice. The way to recreate it - log in and load audit page. Before clicking the OK button of the mission dialog box, click Retake Tutorial. At the end of the tutorial, the mission dialog is shown once from Onboarding.js and next from Main.js.

File: public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js [hunk 2]
```diff
@@ -475,7 +475,15 @@ function Onboarding(svl, actionStack, audioEffect, compass, form, handAnimation,
         var mission = missions[0];
 
         missionContainer.setCurrentMission(mission);
-        modalMission.setMissionMessage(mission, neighborhood);
+        if (missionContainer.isTheFirstMission() && user.getProperty("username") === "anonymous") {
```
Comment (manaswis) [inline public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js]: Could you use the isAnAnonymousUser() method from Main.js? Would enable consistency.

Comment (sagec108) [inline public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js]: I changed this section of code in my latest commit to handle both registered and anonymous users.

File: public/javascripts/SVLabel/src/SVLabel/mission/MissionContainer.js [hunk 3]
```diff
@@ -105,7 +105,9 @@ function MissionContainer (statusFieldMission, missionModel, taskModel) {
             if (missionIds.indexOf(mission.missionId) < 0) self._missionStoreByRegionId[regionId].push(mission);
         }
     };
-
+    this.onlyMissionOnboarding = function (){
```
Comment (manaswis) [inline public/javascripts/SVLabel/src/SVLabel/mission/MissionContainer.js]: Could you rename this to `onlyMissionOnboardingDone`?

Comment (manaswis) [inline public/javascripts/SVLabel/src/SVLabel/mission/MissionContainer.js]: Also add comments

File: public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js (hunk 1 above)
Comment (sagec108) [inline public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js]: I modified the if condition. The mission instructions pop-up after skipping the tutorial, retaking the tutorial, or finishing the regular tutorial. I can't recreate an instance in which the instructions will show up twice. Please let me know if you can recreate this issue in the latest code!

File: public/javascripts/SVLabel/src/SVLabel/mission/MissionContainer.js (hunk 3 above)
Comment (sagec108) [inline public/javascripts/SVLabel/src/SVLabel/mission/MissionContainer.js]: Done!

**Pull request merged**